bpf/kmesh/workload/include/authz.h

```diff
@@ -22,6 +23,25 @@ struct {
     __uint(max_entries, MAP_SIZE_OF_AUTH_POLICY);
 } map_of_authz SEC(".maps");
 
+struct match_result {
+    __u32 action;
+    __u32 match_res;
+    __u16 dport;
+    struct bpf_sock_tuple *tuple_info;
+    void *match;
+};
+
+/*
+ * This map is used to store the variable that
+ * xdp_auth needs to pass during the tail call
+ */
+struct {
+    __uint(type, BPF_MAP_TYPE_ARRAY);
+    __uint(key_size, sizeof(__u32));
+    __uint(value_size, sizeof(struct match_result));
+    __uint(max_entries, 1);
+} map_of_t_data SEC(".maps");
```

Comment:
map_of_tailcall_info

Reply:
> map_of_tailcall_info

this can also be truncated as map_of_tailcall, so I change it to tailcall_info_map